source/payment_links/index.html.md.erb

```diff
@@ -16,5 +16,6 @@ API if your service:
 
 * does not have a development team that can integrate your service with GOV.UK Pay
 
-You can read more about payment links on [the product
-page](https://www.payments.service.gov.uk/payment-links/).
+You can create payment links in English or Welsh.
```

Comment:
I wonder if it will be clear to the reader what it actually means to create a payment link in English or Welsh. Maybe something more along the lines of “Each payment link can be configured to display either English or Welsh instructions to users” (but without the word “configured” because that’s horrible).

Reply:
@alexbishop1 Similar to the above - I'll note this for the 2ier so they can consider.

Reply:
Our 2ier felt this was ok as is.